apps/client/src/views/cuesheet/cuesheet-table/cuesheet-table-settings/CuesheetTableSettings.module.scss

```
@@ -8,6 +8,24 @@
   gap: 1rem;
   font-size: $inner-section-text-size;
   border-radius: 3px 3px 0 0;
+
+  transition: background-color 0.2s ease-in-out;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)